### PR TITLE
[AN-156][AN-122] - nationalities page, clubs page, forms for adding clubs

### PR DESCRIPTION
**Pull request by meg3758**
### Jira ticket

JIRA_LINK
 [AN-156](https://agh-student.atlassian.net/jira/software/c/projects/AN/boards/3/backlog?assignee=712020%3A0b35705e-6294-4def-a844-fb4c33724bdb&selectedIssue=AN-156) 
[AN-122](https://agh-student.atlassian.net/jira/software/c/projects/AN/boards/3/backlog?assignee=712020%3A0b35705e-6294-4def-a844-fb4c33724bdb&selectedIssue=AN-122)

### CODE REVIEW

**Comment by Skalakid** [comment]
Why there are so many changes even in backend?

**Comment by jakubpawlina** [comment]
@Skalakid check now, there was a problem with CRLF, now only significant changes are present

@meg3758 you can try to change CRLF to LF ([helpful discussion](https://stackoverflow.com/questions/1889559/make-git-diff-ignore-m))

File: frontend/sumo-championship/src/pages/AllClubsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        setIsLoading(false);
+      }
+    } catch (e) {
+      console.error(e);
+    }
+  };
+
+  useEffect(() => {
+    getClubs();
+  }, []);
```
Comment (Skalakid) [inline frontend/sumo-championship/src/pages/AllClubsPage.tsx:40]: Fix eslint error

File: frontend/sumo-championship/src/components/organisms/AddClubForm.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      if (!country) setErrorMessage('Country is required');
+        return;
+    }
+    try {
+        const clubInfo: clubInformation = { name, nationality: country };
+        await addClub(clubInfo);
+        setName('');
+        setCountry('');
+        setErrorMessage('');
+        console.log('Club added successfully!');
```
Comment (Skalakid) [inline frontend/sumo-championship/src/components/organisms/AddClubForm.tsx:45]: After adding a club it would be nice to navigate to the all clubs page

Comment (Skalakid) [reply frontend/sumo-championship/src/components/organisms/AddClubForm.tsx:45]: I will handle it in my PR later

**Review by Skalakid — APPROVED** [review]
Good job!

**Pull request merged**